pkg/controller/servingcert/controller/secret_creating_controller.go

```diff
@@ -227,7 +231,20 @@ func (sc *serviceServingCertController) issuedByCurrentCA(secret *corev1.Secret)
 		return false
 	}
 
-	return certs[0].Issuer.CommonName == sc.commonName()
+	if certs[0].Issuer.CommonName != sc.commonName() {
```

Comment:
this is for backwards compatibility? godoc please.

Reply:
I'm assuming you meant add a comment, since godoc applies to declarations? Or did you want a change in godoc of the function?

Reply:
yes

Reply:
Done